sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -148,6 +144,8 @@ def url(self):
 
         This could be either the primary endpoint,
         or the secondary endpoint depending on the current :func:`location_mode`.
+
+        :type: str
```

Comment:
I don't find a doc about it, I added it as it was written in other methods in this class. After re-think, I would like to use `:rtype` as it is actually a return type of this function instead of a type of a param/keyword.